internal/css_parser/css_parser_test.go

```diff
@@ -1013,6 +1013,23 @@ func TestLegalComment(t *testing.T) {
 func TestAtKeyframes(t *testing.T) {
 	expectPrinted(t, "@keyframes {}", "@keyframes \"\" {\n}\n")
 	expectPrinted(t, "@keyframes name{}", "@keyframes name {\n}\n")
+	// string name valid to convert into ident
+	expectPrinted(t, `@keyframes "name"{}`, "@keyframes \"name\" {\n}\n")
+	expectPrintedMinify(t, `@keyframes "name"{}`, "@keyframes \"name\"{}")
+	expectPrintedMangleMinify(t, `@keyframes "name"{}`, "@keyframes name{}")
+
+	// string name valid to convert into ident but need escape
+	expectPrinted(t, `@keyframes "foo bar"{}`, "@keyframes \"foo bar\" {\n}\n")
+	expectPrintedMinify(t, `@keyframes "foo bar"{}`, "@keyframes \"foo bar\"{}")
+	expectPrintedMangleMinify(t, `@keyframes "foo bar"{}`, "@keyframes foo\\ bar{}")
```

Comment:
Add test `@keyframes "none" {}`

Reply:
Because `none` is not included in the definition of `cssWideAndReservedKeywords`.

https://github.com/evanw/esbuild/blob/4a6f64cd81c996aa2868f467ae34e9af46405861/internal/css_parser/css_decls_font_family.go#L24-L34